pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -268,220 +253,175 @@ export class ConfigurationCard extends React.Component {
-                                </td>
-                            </tr>
-
-                            <tr>
-                                <th scope="row">{_("System time")}</th>
-                                <td><ServerTimeConfig /></td>
-                            </tr>
-
-                            <tr>
-                                <th scope="row">{_("Domain")}</th>
-                                <td>{domain_button}</td>
-                            </tr>
-
-                            <tr>
-                                <th scope="row">{_("Performance profile")}</th>
-                                <td><span id="system-info-performance" /></td>
-                            </tr>
-
-                            <tr>
-                                <th scope="row">{_("Secure shell keys")}</th>
-                                <td>
-                                    <Button variant="link" isInline id="system-ssh-keys-link" data-toggle="modal" onClick={this.host_keys_show}
-                                        data-target="#system_information_ssh_keys">{_("Show fingerprints")}</Button>
-                                </td>
-                            </tr>
-
-                            {this.state.pmlogger_switch_visible &&
-                            <tr>
-                                <th scope="row">{_("Store metrics")}</th>
-                                <td>
-                                    <OnOffSwitch
-                                        id="server-pmlogger-switch"
-                                        state={this.pmlogger_service.state === "running"}
-                                        disabled={this.pmlogger_service.state == "starting" || this.state.pm_logger_switch_disabled}
-                                        onChange={this.onPmLoggerSwitchChange} />
-                                </td>
-                            </tr>}
-
-                            {this.state.pcp_link_visible &&
-                            <tr>
-                                <th scope="row">{_("PCP")}</th>
-                                <td>
-                                    <Button isInline variant="link" id="system-configuration-enable-pcp-link" onClick={() => install_dialog("cockpit-pcp")}>
-                                        {_("Enable stored metrics")}
-                                    </Button>
-                                </td>
-                            </tr>}
-
-                        </tbody>
-                    </table>
-                </CardBody>
-            </Card>
+            <>
+                {this.state.hostEditModal && <PageSystemInformationChangeHostname onClose={() => this.setState({ hostEditModal: false })} />}
+                <Card className="system-configuration">
+                    <CardTitle>{_("Configuration")}</CardTitle>
+                    <CardBody>
+                        <table className="pf-c-table pf-m-grid-md pf-m-compact">
```

Comment:
You can also use the react component for that - since you rewrote that whole thing in react. <Table variant="compact" gridBreakPoint="grid-md"

Reply:
oh, I just shifted this block 4 spaces to the right :) Lets do that in some other PR